alert-policies/flutter-android/UsedMemory.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: Used Memory
+
+# Description and details
+description: |+
+  This alert is triggered when the application used memory is above 200 MB for at least 5 mins.
```

Comment:
is it possible to create an alert with percent instead of MB ?

Reply:
I have updated it as per your comments.
Thank you...